### PR TITLE
cert-manager upgrade tweaks

### PR DESCRIPTION
**Pull request by nefischer**

### CODE REVIEW

File: docs/how-to-docs/cert-manager-upgrade-from-psg.md [hunk 1]
```diff
@@ -28,11 +28,13 @@ There are 2 possible approaches for the migration of `Ingress` resources. The hi
 
 For a more detailed understanding of how the manifest files need to be updated, please refer to section [Updating cert-manager resources for v0.13.1](#updating-cert-manager-resources-for-v0131) below.
 
-Option 1 below is strongly recommended as the approach.
+Option 1 is the simplest and is appropriate if you can afford some down-time.
```
Comment (KashifSaadat) [inline docs/how-to-docs/cert-manager-upgrade-from-psg.md:31]: `some down-time` - suggest what that is, e.g. are we talking seconds, minutes, hours potentially?

File: docs/how-to-docs/cert-manager-upgrade-from-v0.8.md [hunk 2]
```diff
@@ -49,11 +49,13 @@ There are 2 possible approaches for the migration of resources. The high-level s
 
 For a more detailed understanding of how the manifest files need to be updated, please refer to section [Updating cert-manager resources for v0.13.1](#updating-cert-manager-resources-for-v0131) below.
 
-Option 1 below is strongly recommended as the approach.
+Option 1 is the simplest and is appropriate if you can afford some down-time.
```
Comment (KashifSaadat) [inline docs/how-to-docs/cert-manager-upgrade-from-v0.8.md:52]: Update also based on previous comment.

File: docs/how-to-docs/cert-manager-upgrade-from-psg.md (hunk 1 above)
Comment (nefischer) [reply docs/how-to-docs/cert-manager-upgrade-from-psg.md]: updated; added:
 (typically a couple of minutes, but longer if there is an issue

File: docs/how-to-docs/cert-manager-upgrade-from-v0.8.md (hunk 2 above)
Comment (nefischer) [reply docs/how-to-docs/cert-manager-upgrade-from-v0.8.md]: updated

**Pull request merged**